config/selectionCfg_ETau_Legacy2016.cfg

```diff
@@ -160,9 +160,9 @@ s0b2jresolvedMcutNoZMCut      = s0b2jMcutNoZMCut, isBoosted != 1
 s0b2jresolvedMcutMtcutNoZMCut = s0b2jresolvedMcutNoZMCut, mT1 < 30
 
 # final categories - Run2 Legacy (including VBF category)
-s1b1jresolvedMcut = baseline , btagM , isBoosted != 1 , ellypsMassCut2, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && bjet1_bID_deepFlavor > 0.3093)
-s2b0jresolvedMcut = baseline , btagMM, isBoosted != 1 , ellypsMassCut2, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && bjet1_bID_deepFlavor > 0.3093)
-sboostedLLMcut    = baseline , btagLL, isBoosted == 1 , boostMassCut2 , !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && bjet1_bID_deepFlavor > 0.3093)
+s1b1jresolvedMcut = baseline , btagM , isBoosted != 1 , ellypsMassCut2, !(isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && (bjet1_bID_deepFlavor > 0.3093 || bjet2_bID_deepFlavor > 0.3093) )
```

Comment:
@francescobrivio sorry for being annoying but why not writing isVBF == 1 && VBFjj_mass > 500 && VBFjj_deltaEta > 3 && bTagMfirst?

Reply:
it should be the same, at this point I don't want to change it again! :)

Reply:
Ok :) but please check if it is correct in the selectionCfg_syst :)

Reply:
in the selection*_syst configs I used bTagMfirst ecc...so they are already correct.